### DISCUSSION TITLE
Keep Your Latest Submission on Gradescope

### DISCUSSION THREAD

**Discussion by spearo2** [prosyslab-classroom/cs424-program-reasoning#459] [Announcements]
안녕하세요,

Gradescope제출 관련하여 당부 말씀 드립니다.
Gradescope에 과제를 제출 하실때, 가장 성적이 높았던 과거 제출물로 노출 점수만 바꾸시지 말고 가장 성적이 높았던 최종 제출물을 새로 업로드 해주세요.
마감기간 이후, 학생분들의 테스트 케이스로 추가 채점할 때, 현재 활성화된 제출물로 채점되지 않을 수 있습니다.

감사합니다.

---

Hello,

When submitting assignments on Gradescope, please do not simply change the active submission to a previous one with the highest score. Instead, please re-upload your final submission that achieved the highest score.

After the deadline, when we evaluate with additional test cases from students, Gradescope could grade your latest submission instead of the currently active score.

Thank you.

**Comment by bongjunj** [comment]
예전 제출물을 다시 제출하면
예전에는 점수가 나왔었는데
새로 업로드하면 실행하다 멈춰서 0점이 나옵니다.

어떻게 하면 좋을까요?

---

If I submit my previous work,
which did show me the execution results as well as the score,
the gradescope fails and gives me zero.

What should I do?
